concepts/units-of-measurement.md

```diff
@@ -221,9 +221,12 @@ SI (base unit symbols):
 | VolumetricFlowRate        | Cubic Metre per Hour             | m³/h   |
 | VolumetricFlowRate        | Cubic Metre per Day              | m³/d   |
 
-<a name="mired-footnote">^</a>: Technically, mireds are the reciprocal of Temperature, but QuantityType and NumberItem will transparently convert between mireds and Kelvin.
+<a name="mired-footnote">^</a>: This unit is used to indicate the color temperature of lights.
+Technically, mireds are the reciprocal of Temperature, but QuantityType and NumberItem will transparently convert between mireds and Kelvin.
 This technicality might be a problem if you are using mireds in rules.
 Be sure to use the `toInvertibleUnit` method on QuantityType, rather than `toUnit`.
+The name 'mired' is a colloquial name and unit symbol for 'micro reciprocal degree'.
+The unit has subsequently been adopted by the SI committee as 'micro reciprocal Kelvin' with the colloquial name 'mirek' and the unit symbol 'MK⁻¹'.
```

Comment:
Do you have a reference for this? https://en.wikipedia.org/wiki/Mired says the unit is `reciprocal megakelvin` which matches more closely with the symbol. Though I believe both terms would be correct - reciprocal megakelvin = "(MK)^-1", while micro reciprocal Kelvin = "u(K^-1)". Though micro reciprocal kelvin is clearly where the portmanteau mirek comes from.

Reply:
It's my best guess. It is 'mirek" rather than 'remek' after all. And mega on the denominator is the same as micro on the numerator. The actual formulae are:
- mirek = 1e6 / kelvin
- kelvin = 1e6 / mirek

Reply:
I asked because the reference on the Wikipedia page is to a printed book.

Reply:
Ok. I tried to improve the text..

Reply:
I guess the best explanation is as follows..

- Original value: 2000 i.e. Kelvin
- Take reciprocal: 1 / 2000 = 0.0005 i.e. Reciprocal Kelvin (too small)
- Scale by one million: 0.0005 * 1000000 = 500 i.e. Micro Reciprocal Kelvin